nmxact/nmp/echo.go

```diff
@@ -22,14 +22,14 @@ package nmp
 import ()
 
 type EchoReq struct {
-	NmpBase
+	NmpBase        `codec:"-"`
 	Payload string `codec:"d"`
 }
 
 type EchoRsp struct {
 	NmpBase
 	Payload string `codec:"r"`
-	Rc      int    `codec:"rc" codec:",omitempty"`
+	Rc      int    `codec:"rc"`
```

Comment:
So you tested this with OIC option, and it works? I don't remember this being always sent with oicmgr. But I could be mistaken.

Reply:
it works because when decoding to structure `omitempty` is not applicable - struct member will just keep its value if no corresponding field is present in stream.